### PR TITLE
[CI Environment] Autofetch ServicePrincipal Object ID

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

| GitHub Pipeline | Azure DevOps |
| - | - |
| [AnalysisServices: Servers](https://github.com/MrMCake/ResourceModules/actions/workflows/ms.analysisservices.servers.yml) | [Build Status](https://dev.azure.com/servicescode/infra-as-code-source/_build/latest?definitionId=2484&branchName=users%2Falsehr%2F1605_spId) |
| [.Platform: Dependencies](https://github.com/MrMCake/ResourceModules/actions/workflows/platform.dependencies.yml) (unrelated) | [Build Status](https://dev.azure.com/servicescode/infra-as-code-source/_build/latest?definitionId=2485&branchName=users%2Falsehr%2F1605_spId) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: .azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml [hunk 1]
```diff
@@ -121,6 +121,13 @@ jobs:
                   $targetFileList += '${{ deploymentBlock.path }}'
                 }
 
+                # Collection additional information
+                ## Service Principal Object ID
```
Comment (eriqua) [inline .azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml:125]: Here and in other occurrences
```suggestion
                # Get Service Principal Object ID
```

Comment (eriqua) [reply .azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml:125]: I'd also suggest to always place this snippet either before or after the `# Get target files` one

File: .github/workflows/platform.dependencies.yml [hunk 2]
```diff
@@ -1250,7 +1248,7 @@ jobs:
             'siterecovery.parameters.json',
             'table.parameters.json',
             'vaultcore.parameters.json',
-            'webpubsub.parameters.json'
+            'webpubsub.parameters.json',
```
Comment (eriqua) [inline .github/workflows/platform.dependencies.yml:1251]: 👻

File: .azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml (hunk 1 above)
Comment (MrMCake) [reply .azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml:125]: Addressed

File: .github/workflows/platform.dependencies.yml (hunk 2 above)
Comment (MrMCake) [reply .github/workflows/platform.dependencies.yml]: done

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**